### PR TITLE
describe_pp: improve error messages

### PR DESCRIPTION
**Pull request by Alizter**
This makes them agnostic to `describe` or `show`.

### CODE REVIEW

File: bin/describe/describe_pp.ml [hunk 1]
```diff
@@ -86,11 +85,10 @@ let get_pped_file super_context file =
         let+ () = Build_system.build_file file_in_build_dir in
         Error file_in_build_dir
       | Some { loc; _ } ->
-        User_error.raise ~loc
-          [ Pp.text "describe pp command doesn\'t work with staged_pps" ])
+        User_error.raise ~loc [ Pp.text "staged_pps are not supported." ])
     | false ->
       User_error.raise
-        [ Pp.textf "%s does not exist" (Path.to_string file_in_build_dir) ])
+        [ Pp.textf "%S does not exist" (Path.to_string file_in_build_dir) ])
```
[the remark this answers is not in the document]
Comment (Alizter) [reply bin/describe/describe_pp.ml:91]: Good point

[the remark this answers is not in the document]
Comment (rgrinberg) [reply bin/describe/describe_pp.ml:91]: Still wrong

File: bin/describe/describe_pp.ml [hunk 2]
```diff
@@ -66,31 +65,32 @@ let get_pped_file super_context file =
...
-                     let preprocess =
-                       Dune_rules.Preprocess.Per_module.(
-                         lib.buildable.preprocess |> single_preprocess)
-                     in
-                     match preprocess with
-                     | Dune_rules.Preprocess.Pps ({ staged = true; _ } as pps)
-                       -> Some pps
-                     | _ -> acc)
-                   | _ -> acc))
+        let open Option.O in
```
Comment (rgrinberg) [inline bin/describe/describe_pp.ml:68]: I don't think this indentation removal gained us anything. It's not even intended very much and the lines have the same wrapping.

Also, don't mix refactoring with real changes. It's hard to review this way.

**Review by rgrinberg — CHANGES_REQUESTED** [review]

File: bin/describe/describe_pp.ml (hunk 2 above)
Comment (Alizter) [reply bin/describe/describe_pp.ml:68]: yeah fair enough, I'll revert this

**Pull request merged**